recipes/selinux/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "selinux" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
```

How about calling this `python-selinux` to avoid confusion?

```suggestion
  name: python-{{ name|lower }}
```

---

Hmm... isn't it better to use the PyPI package names?

---

Conda-forge contains a lot more than just Python packages and as a result can't reasonably have a 1-to-1 mapping to PyPI package naming (even if a lot of packages do match). I don't think there is much harm in prefixing it unless you see an issue?